docs/components/sensor/bme280.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "Configure a bme280 sensor"
+linkTitle: "bme280"
+weight: 70
+type: "docs"
+description: "Configure a bme280 sensor."
+tags: ["sensor", "components"]
+icon: "img/components/sensor.png"
+# SME: #team-bucket
+---
+
+Configure a `bme280` sensor to integrate a [BME280 environmental sensor](https://www.adafruit.com/product/2652), used for sensing temperature, humidity, and barometric pressure, into your robot.
+
+Configure a `bme280` sensor as follows:
+
+{{< tabs >}}
+{{% tab name="Config Builder" %}}
+
+On the **COMPONENTS** subtab of your robot's page in [the Viam app](https://app.viam.com), navigate to the **Create Component** menu.
+Enter a name for your sensor, select the type `sensor`, and select the `bme280` model.
+
+<img src="../img/bme280-sensor-ui-config.png" alt="Creation of a bme280 sensor in the Viam app config builder." style="max-width:800px" />
```

Comment:
Please use \!\[\]\(\) formatting. This doesn't work inside tabs (and I haven't looked into the reason for it)